### DISCUSSION TITLE
Conflicting instructions for installing nvim-orgmode

### DISCUSSION THREAD

**Discussion by wolfv6** [nvim-orgmode/orgmode#764] [Q&A]
I installed nvim-orgmode on LazyVim using the instructions on
https://github.com/nvim-orgmode/orgmode#installation > lazy.nvim (recommended)
It uses:
```
  event = 'VeryLazy',
  ft = { 'org' },
```

The instructions on https://nvim-orgmode.github.io/installation.html say:
Lazy loading (Not recommended)
Lazy loading via ft option works, but not completely. Global mappings are not set because plugin is not initialized on startup.

What are the consequences of not setting global mappings in nvim-orgmode?
Can it cause loss of functionality?
Does "Global mappings" refer to key mappings?

**Comment by rmunn** [comment]
The "Lazy loading (Not recommended)" text is no longer on https://nvim-orgmode.github.io/installation.html, and I can't even find that line in the history of https://github.com/nvim-orgmode/orgmode/tree/master/docs (the source for the nvim-orgmode.github.io site) so I don't know why it changed. But it seems that lazy loading through lazy.nvim is now listed as "(Recommended)" in both places.

The inconsistency between having `ft = { 'org' }` in the README.md instructions vs not having the `ft` line in the website instructions remains, though. And it remains unexplained.
